sync/sync-impl/src/main/res/layout/activity_connect_sync.xml

```diff
@@ -1,5 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
+<?xml version="1.0" encoding="utf-8"?><!--
```

Comment:
what’s with the format change?

Reply:
Not sure. Probably just automatically changed when I formatted the file. @malmstein  should I revert it?